### PR TITLE
CocoaPods 1.0.0

### PR DESCRIPTION
**Pull request by Coeur**
To merge when the time comes.
https://github.com/CocoaPods/CocoaPods/issues/4737

### CODE REVIEW

**Comment by orta** [comment]
Thanks!

**Comment by fulldecent** [comment]
For the people that are testing 1.0.0-beta3 today, could you please advise us on how to use this branch today?

**Comment by orta** [comment]
Use @Coeur's version: `pod lib create https://github.com/Coeur/pod-template/`

**Comment by fulldecent** [comment]
Thank you, I am trying this now. I am not sure what is going on, but can't get my dev module to show up into the example project. Here is the file structure this template created for me.

screen shot 2016-02-04 at 6 24 03 pm

Only change is I replaced REPLACEME with FDWaveformView.swift using XCode.

**Comment by brsunter** [comment]
When I create a project using the https://github.com/Coeur/pod-template/ I get an error saying error: /Users/brian/Library/Developer/Xcode/DerivedData/MyProject-fvvkvycwmwlwpvfnahiftourjuhx/Build/Products/Debug-iphonesimulator/MyProject/MyBundle.bundle: No such file or directory

**Comment by Coeur** [comment]
@brsunter, delete DerivedData, `pod deintegrate`, `pod install`

**Comment by thomasgallagher** [comment]
> Use @Coeur's version: pod lib create https://github.com/Coeur/pod-template/

I ended up using:

`pod lib create --template-url="git://github.com/Coeur/pod-template/" PodName`

**Comment by thomasgallagher** [comment]
Actually - that failed too as there was no description.

PR submitted to Coeur - in the meantime:

`pod lib create --template-url="git://github.com/thomasgallagher/pod-template" PodName`

**Comment by Coeur** [comment]
@thomasgallagher please make an independent pull request as this pull request (#144) is specific to CocoaPods 1.0.0 support. Your change is compatible with CocoaPods 0.39 and could be merged without delay to master branch.

**Comment by Coeur** [comment]
@JaSpa can you help double check that I have correctly resolved the conflict in ProjectManipulator.rb with https://github.com/CocoaPods/pod-template/pull/144/commits/933702934dfb953e7970344134a7e5cf7113a1e4

File: setup/ProjectManipulator.rb [hunk 1]
```diff
@@ -74,6 +74,8 @@ def remove_demo_project
       podfile_path = project_folder + "/Podfile"
       podfile_lines = File.read(podfile_path).lines
       3.times do  podfile_lines.delete_at 2 end
+      podfile_lines.delete_at 4 # inherit-statement
```
Comment (JaSpa) [inline setup/ProjectManipulator.rb]: @Coeur Because there is one line less, we need `podfile_lines.delete_at 3`. Rest looks good.

**Comment by thomasgallagher** [comment]
PR: https://github.com/CocoaPods/pod-template/pull/162

**Comment by Coeur** [comment]
poke @segiddins @orta

**Comment by orta** [comment]
I was just testing it right now actually ;)

**Comment by orta** [comment]
OK, this looks good - thanks for your patient @Coeur !

**Pull request merged**